pkg/client/generated/management/v3/zz_generated_global_role.go

```diff
@@ -37,6 +38,7 @@ type GlobalRole struct {
 	OwnerReferences       []OwnerReference        `json:"ownerReferences,omitempty" yaml:"ownerReferences,omitempty"`
 	Removed               string                  `json:"removed,omitempty" yaml:"removed,omitempty"`
 	Rules                 []PolicyRule            `json:"rules,omitempty" yaml:"rules,omitempty"`
+	Status                *GlobalRoleStatus       `json:"status,omitempty" yaml:"status,omitempty"`
```

Comment:
why is `GlobalRoleStatus` still a pointer here? Maybe you have to generate this file again?

Reply:
So this generated file is for our Norman API, and in order to remove the pointer, I've added some custom logic in `pkg/schemas/management.cattle.io/v3/schema.go`. Note that we don't actually support using Norman with our public API, so users/UI should never create a GlobalRole with a status. But to be safe I added it just in case.